doc/classes/ImageTexture.xml

```diff
@@ -4,10 +4,38 @@
 		A [Texture2D] based on an [Image].
 	</brief_description>
 	<description>
-		A [Texture2D] based on an [Image]. Can be created from an [Image] with [method create_from_image].
-		[b]Note:[/b] The maximum image size is 16384×16384 pixels due to graphics hardware limitations. Larger images will fail to import.
+		A [Texture2D] based on an [Image]. For an image to be displayed, an [ImageTexture] has to be created from it using the [method create_from_image] method:
+		[codeblock]
+		var texture = ImageTexture.new()
+		var image = Image.new()
+		image.load("res://icon.png")
+		texture.create_from_image(image)
+		$Sprite2D.texture = texture
+		[/codeblock]
+		This way, textures can be created at run-time by loading images both from within the editor and externally.
+		[b]Warning:[/b] Prefer to load imported textures over loading them from within the filesystem dynamically, as it may not work in exported projects:
+		[codeblock]
+		var texture = load("res://icon.png")
+		$Sprite2D.texture = texture
+		[/codeblock]
+		This is because [method Image.load] relies on OS-specific file access methods that bypass the virtual filesystem implemented in the engine to load imported resources, so [method Image.load] should only be used in the editor or in cases when you need to load external images at run-time, such as images located at the [code]user://[/code] directory.
```

Comment:
While I agree that using imported StreamTextures should be recommended, it's not clear to me why/how `Image.load()` would fail due to OS-specific file access methods.

Do you mean that it will fail in exported projects in `res://` because the actual PNG file is not in the PCK? If so that's not really OS/file access specific, but due to the file being imported as a StreamTexture and thus not compatible with using as Image.

Reply:
Simplified the rationale that images are imported as StreamTextures. Some information is available in `Image` already.